plugins/candela/web_client/templates/configView.pug

```diff
@@ -6,5 +6,7 @@ p
   |  to render table files directly in Girder.
 
 p.
-  This plugin requires no configuration. Simply upload a CSV file as an item,
-  then set the Candela visualization type and options.
+  This plugin requires no configuration. Simply upload a CSV file as an item
```

Comment:
I'm fixing this in https://github.com/girder/girder/pull/2216 . If this gets merged first, I'll incorporate these documentation changes into that PR. Otherwise, this will need to be rebased.